src/Nethermind/Nethermind.Evm/Tracing/GasEstimator.cs

```diff
@@ -31,6 +36,12 @@ public class GasEstimator
 
         public long Estimate(Transaction tx, BlockHeader header, EstimateGasTracer gasTracer, CancellationToken token = new())
         {
+            return Estimate(tx, header, gasTracer, DefaultErrorMargin, token);
+        }
+        public long Estimate(Transaction tx, BlockHeader header, EstimateGasTracer gasTracer, int errorMargin, CancellationToken token = new())
```

Comment:
Do we need two overloads? Just do `int errorMargin = DefaultErrorMargin`

Reply:
removed